pkg/test/resourcefixture/testdata/basic/redis/v1beta1/basicrediscluster/create.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: redis.cnrm.cloud.google.com/v1alpha1
+kind: RedisCluster
+metadata:
+  name: rediscluster-${uniqueId}
+spec:
+  shardCount: 3
+  pscConfigs:
+  - networkRef:
+      external: projects/${projectId}/global/networks/computenetwork-${uniqueId}
```

Comment:
It's good to mix external and name references for the sake of coverage, though personally I do think the `name` form is much easier :-)